README.md

```diff
@@ -271,7 +195,8 @@ Are you using GitLink in your projects? Let us know and we will add your project
 
 *Note that you can also create a pull request on this document and add it yourself.* 
  
-
 # Icon
 
 Link by Dominic Whittle from The Noun Project
+
+[NuGetDownload]: https://www.nuget.org/packages/gitlink
```

Comment:
I never used this approach. I normally add it directly to the text / link in the content itself. But after your explanation it makes sense.